### DISCUSSION TITLE
Error: cast-wallet-address --private-keys <RAW_PRIVATE_KEYS> Convert a private keys to an address.

### DISCUSSION THREAD

**Discussion by Akash-Kolekar** [foundry-rs/foundry#5493] [Help]
When I run this command:

```
cast wallet address --private-keys 0x1ff690bb679fa3c09173e7e8ab2c1565a53350008933a2f97893a6b4cc2ac52d 0xfe923e23a291aa2020ae9eb2a278e5af009b467a94ffdae2d491b695c34df073
```

I am receiving this error while converting private keys into an address:
```
error: unexpected argument '--private-keys' found
```
but in the documentation the ```--private-keys <RAW_PRIVATE_KEYS>``` command is available.

Can someone help me figure out what the issue is?

**Comment by nathaniel1616** [comment]
You have to convert a single private key to a single address.  The flag '--private-keys' is not found.
In Foundry, the expected flag is "--private-key"  

````bash
cast wallet address --private-key 0x1ff690bb679fa3c09173e7e8ab2c1565a53350008933a2f97893a6b4cc2ac52d
````

Consider this private key as exposed because it is posted on the internet

**Reply by Akash-Kolekar** [reply]
@nathaniel1616 Look at the time when I posted that issue. I think that they have fixed that issue.
